Source/stash.h

```diff
@@ -62,9 +62,9 @@ enum stash_xy_slot : uint8_t {
 struct StashStruct {
 	void RemoveStashItem(int iv, bool calcScrolls);
 
-	Item StashList[100];
-	int8_t StashGrid[NUM_STASH_GRID_ELEM];
-	int _pNumStash;
+	std::vector<Item> StashList;
+	std::vector<int8_t> StashGrid;
+	int NumStash;
```

Comment:
Only globals start with upper case:
```suggestion
	std::vector<Item> stashList;
	std::vector<int8_t> stashGrid;
	int numStash;
```